.tree-sitter-lint/tree-sitter-lint-local/src/bin/tree-sitter-lint-local-lsp.rs

```diff
@@ -0,0 +1,5 @@
+use tree_sitter_lint::tokio;
```

Comment:
I manually ran `rustfmt` on these generated files just so they'd be easier to read here (since they're "new")